test/end-to-end/reports/stock_exit/stock_exit.spec.js

```diff
@@ -0,0 +1,50 @@
+const chai = require('chai');
+const helpers = require('../../shared/helpers');
+
+helpers.configure(chai);
```

Comment:
I only discovered this recently but ... we don't need to call `helpers.configure(chai)` in every module.  It will be configured globally in our protractor.conf.js.  So you can remove this line :)